pkg/virt-handler/migration-proxy/migration-proxy_test.go

```diff
@@ -151,7 +151,7 @@ var _ = Describe("MigrationProxy", func() {
 
 				Expect(err).ShouldNot(HaveOccurred())
 
-				manager := NewMigrationProxyManager(tmpDir, tlsConfig)
+				manager := NewMigrationProxyManager(tmpDir, tlsConfig, tlsConfig)
```

Comment:
i know this is annoying, you might want to make unique server/client tls configs here so we don't screw this up later on and get them reversed.